src/compiler/compile/nodes/Element.ts

```diff
@@ -70,6 +70,47 @@ const a11y_labelable = new Set([
 	'textarea'
 ]);
 
+const a11y_nested_implicit_semantics = new Map([
+	['header', 'banner'],
+	['footer', 'contentinfo']
+]);
+
+const a11y_implicit_semantics = new Map([
+	['a', 'link'],
+	['article', 'article'],
```

Comment:
I think this can be removed.

Reply:
No problem - many requested changes is good for me learning and the contribution process :)

I've made the change in 6ab5c7fa6fdc0f75a5c82f89d801f80c00728284